docs/src/tutorials/tutorial_1.md

```diff
@@ -5,59 +5,70 @@ This tutorial walks through the steps to download, run, and view the output of a
+The next step is to run FUND. If you wish to first get more acquainted with the model itself, take a look at the provided online [documentation](http://www.fund-model.org).  
 
-In order to run FUND, you will need to have the packages `Distributions` and `StatsBase` installed, and if not do so by entering [Pkg mode](https://docs.julialang.org/en/v1/stdlib/Pkg/index.html) by typing `]`, and then `add StatsBase` and `add Distributions`.  
+Now open a julia REPL and type the following command to load the MimiFUND package into the current environment: 
 
-Now open a julia REPL and type 
-
-```
+```jldoctest tutorial1; output = false
 using MimiFUND
-```
-to access the public API to FUND, which currently includes the function `getfund`, a function that returns a version of fund allowing for different user specifications.  
+
+# output
 
 ```
-m = getfund()
+Now we can access the public API of FUND, including the function `MimiFUND.get_model`. This function returns a copy of the default FUND model. Here we will first get the model, and then use the `run` function to run it.
+
+```jldoctest tutorial1; output = false, filter = r".*"s
+m = MimiFUND.get_model()
 run(m)
-```
 
-Note that these steps should be relatively consistent across models, where a repository for `ModelX` should contain a primary file `ModelX.jl` which exports, at minimum, a function named something like `getModelX` or `construct_ModelX` which returns a version of the model, and can allow for model customization within the call.
+# output
 
-In this case, the function `getfund` has the signature
-``` 
-getfund(; nsteps = default_nsteps, datadir = default_datadir, params = default_params)
 ```
-Thus there are no required arguments, although the user can input `nsteps` to define the number of timesteps (years in this case) the model runs for, `datadir` to define the location of the input data, and `params`, a dictionary definining the parameters of the model.  For example, if you wish to see only the first 100 timesteps,you may use:
+
+Note that these steps should be relatively consistent across models, where a repository for `ModelX` should contain a primary file `ModelX.jl` which exports, at minimum, a function named something like `get_model` or `construct_model` which returns a version of the model, and can allow for model customization within the call.
+
+In the MimiFUND package, the function `get_model` has the signature
+```julia
+get_model(; nsteps = default_nsteps, datadir = default_datadir, params = default_params)
 ```
+Thus there are no required arguments, although the user can input `nsteps` to define the number of timesteps (years in this case) the model runs for, `datadir` to define the location of the input data, and `params`, a dictionary definining the parameters of the model.  For example, if you wish to run only the first 200 timesteps, you may use:
+```jldoctest tutorial1; output = false, filter = r".*"s
```

Comment:
why is this `output = false` here? Don't we want to make sure the model runs without error?